client/lib/invites/actions.js

```diff
@@ -8,46 +8,46 @@ import { get } from 'lodash';
 /**
  * Internal dependencies
  */
-import Dispatcher from 'calypso/dispatcher';
 import wpcom from 'calypso/lib/wp';
-import { action as ActionTypes } from 'calypso/lib/invites/constants';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { errorNotice, successNotice } from 'calypso/state/notices/actions';
 import { acceptedNotice } from 'calypso/my-sites/invites/utils';
 import { requestSites, receiveSites } from 'calypso/state/sites/actions';
 import { requestSiteInvites } from 'calypso/state/invites/actions';
+import { INVITE_ACCEPTED } from 'calypso/state/action-types';
 
 /**
  * Module variables
  */
 const debug = new Debug( 'calypso:invites-actions' );
 
-export function createAccount( userData, invite, callback ) {
-	const send_verification_email = userData.email !== invite.sentTo;
+function inviteAccepted( invite ) {
+	return { type: INVITE_ACCEPTED, invite };
+}
 
+export function createAccount( userData, invite ) {
```

Comment:
Now that we have a `inviteAccepted` action creator and thunks for invitation actions, should all this be moved to the `state/invites` directory? We [already have](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/invites/actions.js#L35) some action creators and thunks in `actions.js` there.

Reply:
Pushed a commit with the move :+1: I originally didn't want to do it in order to not mess up the diff too much. But now when the review is in advanced stage, let's do it.